internal/api/deployments.go

```diff
@@ -82,40 +82,3 @@ type DeploymentUpdate struct {
 // {"deployments": {"handle": {"any_": ["test"]}}}.
 type DeploymentFilter struct {
 }
-
-type DeploymentAccess struct {
-	BaseModel
-	AccountID     uuid.UUID               `json:"account_id"`
-	WorkspaceID   uuid.UUID               `json:"workspace_id"`
-	DeploymentID  uuid.UUID               `json:"deployment_id"`
-	AccessControl DeploymentAccessControl `json:"access_control"`
-}
-
-// DeploymentAccessSet is a subset of DeploymentAccess used when Setting deployment access control.
-type DeploymentAccessSet struct {
-	AccessControl DeploymentAccessControlSet `json:"access_control"`
-}
-
-// DeploymentAccessControlSet is a definition of deployment access control.
-type DeploymentAccessControlSet struct {
-	ManageActorIDs []string `json:"manage_actor_ids"`
-	RunActorIDs    []string `json:"run_actor_ids"`
-	ViewActorIDs   []string `json:"view_actor_ids"`
-	ManageTeamIDs  []string `json:"manage_team_ids"`
-	RunTeamIDs     []string `json:"run_team_ids"`
-	ViewTeamIDs    []string `json:"view_team_ids"`
-}
-
-// DeploymentAccessControl is a definition of deployment access control.
-type DeploymentAccessControl struct {
-	ManageActors []Actor `json:"manage_actors"`
-	RunActors    []Actor `json:"run_actors"`
-	ViewActors   []Actor `json:"view_actors"`
-}
-
-type Actor struct {
-	ID    uuid.UUID `json:"id"`
-	Name  string    `json:"name"`
-	Email string    `json:"email"`
-	Type  string    `json:"type"`
-}
```

Comment:
Moved this to a separate file for clarity. We don't have a consistent pattern for this yet - for example, Workspaces have this more separated while Blocks have it together (in some places) - open to personal opinions

Reply:
good call